appConfig/appConfig.snd3.yaml

```diff
@@ -0,0 +1,9 @@
+- key: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
Check with Asad if snd4 is availble?

Reply:
Checked with Asad, it is not 100% ready, so not included now